assets/js/modules/adsense/util/is-zero-report.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Report utilities.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Checks whether the report data is empty or not.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} report Report data.
+ * @return {boolean} TRUE if the report has no data, otherwise FALSE.
+ */
+export function isZeroReport( report ) {
+	if ( typeof report === 'undefined' ) {
```

Comment:
Very minor, but we've been tending toward `report === undefined` instead of using `typeof`. I know back in the day `typeof` was more used, but no one redefines `undefined` in ES6+ and it's a bit cleaner to read.

```suggestion
	if ( report === undefined ) {
```